lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -612,6 +612,18 @@ abstract class PrimaryRoleManager {
     for (final RoleManager secondaryRole in secondaryRoles) {
       secondaryRole.update();
     }
+
+    if (semanticsObject.isIdentifierDirty) {
+      _updateIdentifier();
+    }
+  }
+
+  void _updateIdentifier() {
+    if (semanticsObject.hasIdentifier) {
+      setAttribute('semantics-identifier', semanticsObject.identifier!);
```

Comment:
I wonder if it would be better to prefix it with `data-`?   
[https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*)